pkgs/development/tools/misc/sysbench/default.nix

```diff
@@ -33,11 +32,28 @@ stdenv.mkDerivation rec {
     # The bundled version does not build on aarch64-darwin:
     # https://github.com/akopytov/sysbench/issues/416
     "--with-system-luajit"
-    "--with-system-ck"
     "--with-mysql-includes=${lib.getDev libmysqlclient}/include/mysql"
     "--with-mysql-libs=${libmysqlclient}/lib/mysql"
   ];
 
+  # Vendored libck version require more hacks than regular nixpkgs' one.
+  # Sysbench related on statically linked vendored libck.
+  postPatch = ''
+    substituteInPlace \
+      third_party/concurrency_kit/ck/configure \
+        --replace \
+          'COMPILER=`./.1 2> /dev/null`' \
+          "COMPILER=gcc" \
```

Comment:
I'm maybe missing something here; is there a reason `COMPILER` is being set to `gcc` instead of `$CC` or `${stdenv.cc.targetPrefix}cc`?

Reply:
Fixed.

Reply:
We support mingw64 as well, but it's very experimental, so I don't think we need to fix sysbench for it just now.

Reply:
Mhm. Actually this doesn't work for cross compiling.